### PR TITLE
[DEV] add random cost with test

### PR DESCRIPTION
**Pull request by leclere**

### CODE REVIEW

File: src/oneStepOneAleaProblem.jl [hunk 1]
```diff
@@ -52,12 +52,34 @@ function solve_one_step_one_alea(model,
...
+    #update objective
+    if isa(model.costFunctions, Function)
+        try
+            @objective(m, Min, model.costFunctions(t, x, u, xi) + alpha)
+        catch
+            #FIXME: hacky redefinition of costs as JuMP Model
+            @objective(m, Min, model.costFunctions(m, t, x, u, xi) + alpha)
+        end
+
+    elseif isa(model.costFunctions, Vector{Function})
```
Comment (trigaut) [inline src/oneStepOneAleaProblem.jl:73]: I guess there will be an issue in this case as you are adding constraints on cost without removing the previous ones (for the other values of w)

File: src/oneStepOneAleaProblem.jl [hunk 2]
```diff
@@ -52,12 +52,34 @@ function solve_one_step_one_alea(model,
...
     w = getvariable(m, :w)
     alpha = getvariable(m, :alpha)
 
-    # Update value of w:
-    setvalue(w, xi)
+    # Update value of w: 
+    for i in 1:model.dimNoises
+        JuMP.fix(w[i], xi[i])
+    end
+
```
Comment (trigaut) [inline src/oneStepOneAleaProblem.jl]: It's a detail but you can do `JuMP.fix.(w, xi)` instead of the for loop

**Review by trigaut — COMMENTED** [review]
We should prevent travis from testing the package on julia0.4. 

I think the only issue of this PR is the redefinition of the objective when the cost function is polyhedral

File: src/oneStepOneAleaProblem.jl (hunk 1 above)
Comment (leclere) [reply src/oneStepOneAleaProblem.jl:73]: Good point. Have to think about it.

Comment (trigaut) [reply src/oneStepOneAleaProblem.jl:73]: Maybe we should decide not to redefine cost in the piecewise linear case until we are able to handle modifications of constraints LHS

**Pull request closed without merging**